insights/config/specs.py

```diff
@@ -94,6 +94,7 @@
     "docker_sysconfig"          : DockerHostSimpleFileSpec("etc/sysconfig/docker"),
     "dumpe2fs-h"                : CommandSpec("/sbin/dumpe2fs -h {dumpdev}", dumpdev=r"\S+"),
     "engine.log"                : SimpleFileSpec("var/log/ovirt-engine/engine.log", large_content=True),
+    "engine-config_all"         : CommandSpec("/usr/bin/engine-config --all"),
```

Comment:
In the title and description, you wrote command `ovirt-engine --all`, but here, it's `engine-config --all`. I don't know if they are the same, but would you please to explain a bit?

Reply:
Done